lib/players/computer.rb

```diff
@@ -0,0 +1,131 @@
+require 'pry'
+class Computer < Player
+
+  def move(board)
+    valid_moves = []
+    new_board = Array.new(9, " ") 
+    board.cells.each.with_index do |cell, index|
+      valid_moves << index if board.valid_move?(index)
+    end
+
+    if board.cells == new_board
+      9
+    elsif board.cells == ["X", " ", " ", " ", " ", " ", " ", " ", " "]
+      5
+    elsif board.cells == [" ", "X", " ", " ", " ", " ", " ", " ", " "]
+      5
+    elsif board.cells == [" ", " ", "X", " ", " ", " ", " ", " ", " "]
+      5
+    elsif board.cells == [" ", " ", " ", "X", " ", " ", " ", " ", " "]
+      5
+    elsif board.cells == [" ", " ", " ", " ", "X", " ", " ", " ", " "]
+      9
+    elsif board.cells == [" ", " ", " ", " ", " ", "X", " ", " ", " "]
+      5
+    elsif board.cells == [" ", " ", " ", " ", " ", " ", "X", " ", " "]
+      5
+    elsif board.cells == [" ", " ", " ", " ", " ", " ", " ", "X", " "]
+      5
+    elsif board.cells == [" ", " ", " ", " ", " ", " ", " ", " ", "X"]
+      5
```

Comment:
Couldn't these 9 `elsif` statements be condensed into `elsif board.cells.count("X") == 1`